sdk/core/core-http/lib/serviceClient.ts

```diff
@@ -613,6 +613,103 @@ function createDefaultRequestPolicyFactories(
   return factories;
 }
 
+export function createPipelineFromOptions(
+  pipelineOptions: InternalPipelineOptions,
+  authPolicyFactory?: RequestPolicyFactory
+) : ServiceClientOptions {
+  let requestPolicyFactories: RequestPolicyFactory[] = [];
+
+  let userAgentPrefix = undefined;
```

Comment:
Rename this to `userAgentValue`.
It gets passed to policy which expects the value not a prefix

Reply:
Renamed